spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'logger'
+
+describe ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm do
+  let(:ems) do
+    FactoryBot.create(:ems_ibm_cloud_vpc, :provider_region => "us-east").tap do |ems|
+      ems.authentications << FactoryBot.create(:authentication, :auth_key => 'IBMCVS_API_KEY')
+    end
+  end
+
+  let(:vm) do
+    FactoryBot.create(:vm_ibm_cloud_vpc, :ext_management_system => ems)
+  end
+
+  context "is_available?" do
+    let(:power_state_on)        { "running" }
+    let(:power_state_suspended) { "stopped" }
+
+    context "with :start" do
+      let(:state) { :start }
+      include_examples "Vm operation is available when not powered on"
+    end
+
+    context "with :stop" do
+      let(:state) { :stop }
+      include_examples "Vm operation is available when powered on"
+    end
+
+    context "with :reboot_guest" do
+      let(:state) { :reboot_guest }
+      include_examples "Vm operation is available when powered on"
+    end
+
+    context "with :reset" do
+      let(:state) { :reset }
+      include_examples "Vm operation is available when powered on"
+    end
+  end
+
+  describe 'power operations' do
+    before(:each) do
+      allow(vm).to receive(:provider_object).and_return(instance)
+    end
+
+    let(:parent) do
+      vpc = double("IBM::Cloud::SDK::Vpc")
+      allow(vpc).to receive(:logger).and_return(Logger.new($stdout))
+      allow(vpc).to receive_messages(:url => nil, :token => nil, :connection => nil)
+      vpc
+    end
+
+    let(:actions) do
+      actions = IBM::Cloud::SDK::VPC::INSTANCE::Actions.new(parent)
```

Comment:
Just need to require the sdk gem here and below before using it then the specs should be green